src/Infrastructure/Processing/AssetProcessingJobPayload.php

```
@@ -0,0 +1,84 @@
+{
+    private const ASSET_ID_FIELD = 'assetId';
+    private const INITIAL_RETRY_COUNT = 0;
+    private const RETRY_COUNT_FIELD = 'retryCount';
+
+    private string $assetId;
+    private int $retryCount;
+
+    public function __construct(string $assetId, int $retryCount)
+    {
+        if ($retryCount < 0) {
+            throw new \InvalidArgumentException('Retry count must be zero or greater.');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '21', column '23'). (undefined)

(MissingImport)